spec/system/admin/users_spec.rb

```diff
@@ -136,16 +136,32 @@
         expect(page).to have_no_text "Email confirmation is pending"
       end
 
+      it "uses the instance default locale for new new user" do
+        visit spree.new_admin_user_path
+
+        expect(page).to have_select('Language', selected: 'English')
+      end
```

Comment:
This spec can be part of the next. While it's generally good to have separate `it` blocks for different expectations, system specs are slow and combining them helps reduce spec run time (and therefore CO2 emisions as well).

Reply:
Thank you 👍  done by you in commit [997c8a4](https://github.com/openfoodfoundation/openfoodnetwork/pull/11487/commits/997c8a49ca2dfb54039c1cc442f8404fdaeec850)